arangod/Cluster/ActionDescription.h

```diff
@@ -65,6 +65,7 @@ struct ActionDescription final {
   ActionDescription(
       std::map<std::string, std::string> description,
       int priority,
+      bool forced,
```

Comment:
I think the boolean parameter is hard to read at the call sites. Something like an `enum class DuplicateActionHandling { DONT_EXECUTE_DUPLICATE_ACTIONS, ALWAYS_EXECUTE_ACTIONS };` or something like that. Actually while writing this I realized I didn't fully understand what the flag does, so it really would help.

(I think the same is true for `int priority`, and a `struct Priority { int value; }` would help there, too)